### ISSUE TITLE
Lack of support for adding elements to mv-item-bar

### ISSUE THREAD

**Issue by synicalsyntax** [mavoweb/mavo#700]
[Example](https://jsfiddle.net/vcLh7sz6/3/)

Two major problems discovered while attempting to add additional elements to be displayed in mv-item-bar:
  * Styling gets applied to buttons only, which is inconsistent with the support for `.mv-button` elements in the toolbar
  * Even after wrapping elements inside parent button, it gets shown by default until first edit (something wrong with initialization)

Reviewing `src/ui.itembar.js` and `mavo.scss` ([line](https://github.com/mavoweb/mavo/blob/dbbaea35fc96c6eb9bc955430953737d2157fa80/src-css/mavo.scss#L53)) and their hardcoded nature for all 3 default buttons demonstrates the lack of support of customization.

**Comment by LeaVerou** [comment]
Yeah, item bars need an overhaul in general to be as extensible as the Mavo toolbar.

One of the reasons this was not done is that it's partially blocked on #72.
